### PR TITLE
Add docs.gl to resources

### PR DESCRIPTION
**Pull request by bubba2k**
docs.gl is a super nifty website I wouldn't want to go without when fiddling with OpenGL.

### CODE REVIEW

File: resources/index.html [hunk 1]
```diff
@@ -44,6 +44,7 @@
 		* [Guide to Modern OpenGL Functions](https://github.com/fendevel/Guide-to-Modern-OpenGL-Functions)
 		* [Learn OpenGL](https://learnopengl.com/)
 		* [OpenGL 4.6 Specification](https://www.khronos.org/registry/OpenGL/specs/gl/glspec46.core.pdf)
+		* [docs.gl - OpenGL API Documentation](http://docs.gl/)
```
Comment (nickclark2016) [inline resources/index.html]: We have the resources in alphabetical order for easy lookup.

**Review by nickclark2016 — CHANGES_REQUESTED** [review]

**Comment by bubba2k** [comment]
Done!

**Review by deccer — APPROVED** [review]
Looks good to me now.

**Review by nickclark2016 — APPROVED** [review]

**Comment by nickclark2016** [comment]
Approved and merging.

**Pull request merged**